src/render_keyhook_camera.c

```diff
@@ -26,17 +26,20 @@ void	ft_keyhook_rot_cam(int key, t_vec3 *direction)
 
 void	ft_keyhook_move_cam(int key, t_cam *cam)
 {
-	if (key == XK_q)
+	t_interval	lim;
+
+	lim = (t_interval){.min = FLOAT_MIN, .max = FLOAT_MAX};
```

Comment:
Not sure, but maybe we are allowed to do all this with declaration in one line? Need to check with norminette

Reply:
True! I made it const and norminette is accepting of it. Also changed the other one in keyhook_utils